pkgs/development/python-modules/certbot/default.nix

```diff
@@ -72,8 +71,6 @@ buildPythonPackage rec {
     "-W ignore::DeprecationWarning"
   ];
 
-  doCheck = true;
```

Comment:
I see tests are still executed - I guess `doCheck` became the default, or it is the default for `setuptools` projects :+1:

Reply:
It's set by `buildPythonPackage`.